### ISSUE TITLE
cannot read required tag data

### ISSUE THREAD

**Issue by ExtraHash** [turtlecoin/turtlecoin-utils#37]
I get this scanning (I believe) the coinbase transaction in block 501848.

To reproduce:

```
import { Transaction } from "turtlecoin-utils";

const rawTx = '010001026406da9321e67d01010101146a70fd9740d18f8999ac2caaa85eab0096f2ddfe0f2b978f7b97da22950f360301022f35effe7587ddbf7210675a222ed28e4c5109c0fb129d24348575e8f628ecc10902e51afe3cb7f9d64b92e871420f30d92c21100a19ef1bfb5d767f769783a1f10d5002e86db3c13f03e06bdeb83828809a271961bea5483dd14bdefa3d773b77f5eb82da0c019568375d0fd72b8435042c7b0da2fe06f72556d72659bbbcba0866445a8e080702340600000001000000ffffffff01000000000000000401000000e20153797374656d2e436f6c6c656374696f6e732e47656e657269632e44696374696f6e61727960325b5b53797374656d2e4f626a6563742c206d73636f726c69622c2056657273696f6e3d342e302e302e302c2043756c747572653d6e65757472616c2c205075626c69634b6579546f6b656e3d623737613563353631393334653038395d2c5b53797374656d2e4f626a6563742c206d73636f726c69622c2056657273696f6e3d342e302e302e302c2043756c747572653d6e65757472616c2c205075626c69634b6579546f6b656e3d623737613563353631393334653038395d5d040000000756657273696f6e08436f6d7061726572084861736853697a650d4b657956616c756550616972730003000308910153797374656d2e436f6c6c656374696f6e732e47656e657269632e4f626a656374457175616c697479436f6d706172657260315b5b53797374656d2e4f626a6563742c206d73636f726c69622c2056657273696f6e3d342e302e302e302c2043756c747572653d6e65757472616c2c205075626c69634b6579546f6b656e3d623737613563353631393334653038395d5d08e60153797374656d2e436f6c6c656374696f6e732e47656e657269632e4b657956616c75655061697260325b5b53797374656d2e4f626a6563742c206d73636f726c69622c2056657273696f6e3d342e302e302e302c2043756c747572653d6e65757472616c2c205075626c69634b6579546f6b656e3d623737613563353631393334653038395d2c5b53797374656d2e4f626a6563742c206d73636f726c69622c2056657273696f6e3d342e302e302e302c2043756c747572653d6e65757472616c2c205075626c69634b6579546f6b656e3d623737613563353631393334653038395d5d5b5d0100000009020000000300000009030000000402000000910153797374656d2e436f6c6c656374696f6e732e47656e657269632e4f626a656374457175616c697479436f6d706172657260315b5b53797374656d2e4f626a6563742c206d73636f726c69622c2056657273696f6e3d342e302e302e302c2043756c747572653d6e65757472616c2c205075626c69634b6579546f6b656e3d623737613563353631393334653038395d5d00000000070300000000010000000100000003e40153797374656d2e436f6c6c656374696f6e732e47656e657269632e4b657956616c75655061697260325b5b53797374656d2e4f626a6563742c206d73636f726c69622c2056657273696f6e3d342e302e302e302c2043756c747572653d6e65757472616c2c205075626c69634b6579546f6b656e3d623737613563353631393334653038395d2c5b53797374656d2e4f626a6563742c206d73636f726c69622c2056657273696f6e3d342e302e302e302c2043756c747572653d6e65757472616c2c205075626c69634b6579546f6b656e3d623737613563353631393334653038395d5d0c05000000404d657373656e6765722c2056657273696f6e3d312e302e302e302c2043756c747572653d6e65757472616c2c205075626c69634b6579546f6b656e3d6e756c6c04fcffffffe40153797374656d2e436f6c6c656374696f6e732e47656e657269632e4b657956616c75655061697260325b5b53797374656d2e4f626a6563742c206d73636f726c69622c2056657273696f6e3d342e302e302e302c2043756c747572653d6e65757472616c2c205075626c69634b6579546f6b656e3d623737613563353631393334653038395d2c5b53797374656d2e4f626a6563742c206d73636f726c69622c2056657273696f6e3d342e302e302e302c2043756c747572653d6e65757472616c2c205075626c69634b6579546f6b656e3d623737613563353631393334653038395d5d02000000036b65790576616c7565020206060000000543616e7469090700000005070000001c547572746c65436f696e2e4d6573736167696e672e4d657373616765030000001a3c54696d657374616d703e6b5f5f4261636b696e674669656c64173c53656e6465723e6b5f5f4261636b696e674669656c64183c436f6e74656e743e6b5f5f4261636b696e674669656c640001010d05000000e88447d10dc8d508090600000006090000000b436865636b20636865636b0b9888df3c34a6c1f8fdbb7720dc3160964b2877c16472a78138aa46a75ceed30683f2c8e959292d508df084b33064e5a9fb4006bc4c871ca15563787e3b047a01c03dadd692db2779a66d0a27c431624e45adb932520e28186db64ceda3c4510f8c924e8e58ffb9dd75559b0abf4accd40198c72f530f736678bf14dbe790160ede44a0e9c51fa41d41c6486613afdfb1d087c2142d08aeb9f8225b798e66e40fbb25a9f390730c058f326a75f52a05f7f2343dab58a0fcde22d9dfb27634ba05b3b26c24b4ab969e0f03a00324fce81dfed1bb9f0d69869c44e08895895af007bc689a51a86893324a1ba6d34f419da794640fadf48804690ebfc56fa1de9d0fa161e3837a7fe5ce474849e65741966fd1bd212bb85029bc395abb8ef09c4e0094671ab056107a3abd2e918541991885f3c236ac12d571fa740d2d4abe94220f49aa8b6093d79d0128cd6ede4ba0ba03ccf0e099509632c8ed3cc77d395f290bdd359224921fe0fbb093f8df87814732c73060ff46bc7289abd05ee5422a0d0d';

const tx = Transaction.from(rawTx);

console.log(tx);
```

**Comment by brandonlehmann** [comment]
Please confirm blockhash.

**Comment by brandonlehmann** [comment]
Believed to be resolved in 4ebcb735b58911acc1f4c0b4ee77be440b5cd441
